markdown/ep-lifecycle/ep-lifecycle.md

```diff
@@ -0,0 +1,151 @@
+author: jessemenning
+summary: See how Event Portal can guide you on a smooth path from fantastic idea to the hands of your users
+id: ep-lifecycle
+tags:
+categories: eventportal
+environments: Web
+status: Hidden
+feedback link: https://github.com/SolaceDev/solace-dev-codelabs/blob/master/markdown/ep-lifecycle
+
+# Event Portal: event-driven innovation that's faster, easier, more resilient
+
+## What you'll learn: Overview
+
+Duration: 0:05:00
+
+Enter a codelab overview here: what & why and github repo link where you can find related code if applicable
+
+## What you need: Prerequisites
+
+Duration: 0:07:00
+
+**For all sections, you'll need to:**
+- Complete the *Connecting Event Portal to a runtime Event Broker* code lab
+
+**For config push and CICD integration sections, you will also need to:**
+- Have a running Event Management Agent connected to an Event Broker
+- A GitHub account.  If you don’t have one, you can [get one free](https://github.com/signup). 
+
+## Discover event-driven stuff you didn’t know existed
+### Find interesting events to use in your solution
```

Comment:
We'll need a `Duration: 0:07:00` line under the  `## Discover event-driven stuff you didn’t know existed` title

Reply:
added